docs/files.rst

```diff
@@ -59,7 +59,7 @@ Start by inserting django-tenants' ``django_tenants.staticfiles.finders.TenantFi
     STATICFILES_FINDERS = [
         "django_tenants.staticfiles.finders.TenantFileSystemFinder",  # Must be first
         "django.contrib.staticfiles.finders.FileSystemFinder",
-        "django.contrib.staticfiles.finders.AppDirectoriesFinder",
+        "django_tenants.staticfiles.finders.TenantAppDirectoriesFinder", # This replaces Django's AppDirectoriesFinder
```

Comment:
Instead of removing the standard ``AppDirectoriesFinder``, consider just inserting ``TenantAppDirectoriesFinder`` above it.

This way, static file lookups can fall back to using the Django standard finders if there are no tenant-specific files to serve.

Reply:
The `TenantAppDirectoriesFinder` is just a wrapper on top of the `AppDirectoriesFinder`, so it doesn't really matter. Even if I leave the `AppDirectoriesFinder`, it won't trigger ever, as the `TenantAppDirectoriesFinder` will find whatever `AppDirectoriesFinder` is supposed to find.